packages/components/Link/src/Link.tsx

```diff
@@ -32,6 +30,15 @@ export function useAsLink(userProps: IWithLinkOptions<IViewProps>): ILinkHooks {
   );
 
   const pressable = useAsPressable({ onPress: linkOnPress, ...rest });
+  const onKeyDown = React.useCallback(
```

Comment:
Are we sure it should be onKeyDown instead of onKeyUp? I feel more often, the "action" happens on key up. Maybe I'm wrong for link though

Reply:
Thanks for pointing that out! I think I implemented the same for Button. I'll make the fixes